core/validation/pre_transformation_validate.py

```diff
@@ -18,33 +25,44 @@ def pre_transformation_validations(
 ):
     """Performs pre-transformation validations based on the provided schema.
 
+    The submission form structures of TF Round 1 & Round 2 do not permit pre-transformation validations
+    as they have already been transformed.
+    Authorisation was introduced in TF Round 4 in order to ensure that users did not submit for local authorities
+    they do not belong to, and does not occur for previous TF rounds.
+    wrong_input_validation occurs first as we cannot authorise user input if it is invalid.
+    conflicting_input_validation occurs next as we cannot authorise user input if it is contradictory.
+
     :param workbook: A dictionary where keys are sheet names and values are pandas
                      DataFrames representing each sheet in the submission.
     :param reporting_round: The reporting round for which the validations are performed.
-    :param auth: A dictionary containing authorization information.
+    :param auth: A dictionary containing authorisation information.
     :raises ValidationError: If any of the validation functions catch a validation error.
     """
     if reporting_round in [1, 2]:
-        return  # do not do pre-transformation validation on historical data
+        return
 
     schema = REPORTING_ROUND_TO_PRE_TRANSFORMATION_SCHEMA[reporting_round]
 
-    wrong_input_validation(workbook, schema)  # first, check if valid
+    wrong_input_validation(workbook, schema)
     conflicting_input_validation(workbook, schema)
 
     if reporting_round == 4 and auth:
         authorisation_validation(workbook, auth, schema)
-        # check last as do not want raise authorisation error if input is invalid or conflicting
 
 
 def authorisation_validation(workbook: dict[str, pd.DataFrame], auth: dict, schema: dict):
-    """Performs authorization validation based on the provided schema.
+    """Performs authorisation validation based on the provided schema.
+
+    An 'auth' dictionary is sent to the /ingest API when the user attempts to submit.
+    This dictionary contains variables against which the submission form is checked
+    to ensure that the user is able to submit the selected information based on data
+    collected from the frontend of the application.
```

Comment:
"is able to submit the selected information based on their user level permissions, which is handled by the frontend" may be more clear?